src/main/java/com/zombiemike/App.java

```diff
@@ -1,4 +1,4 @@
-package com.zombimike;
+package com.zombiemike;
 import java.io.*;
 import java.nio.file.*;
```

Comment:
En annan "bra sak" att göra är att undvika att använda sig av *-importering! Detta kan du ställa in i IntelliJ så den inte kollapsar imports till *